templates/mobile-awareness/decorators/decorateAppPage.js

```diff
@@ -0,0 +1,164 @@
+ * governing permissions and limitations under the License.
+ */
+
+// import {
+//   createTag,
+// } from '../../../pages/scripts/scripts.js';
+
+import { createTag } from '../../../pages/scripts/scripts.js';
+import {
+  hoist, openModalLink, replaceTag, wrapContents,
+} from '../scripts/scripts.js';
+
+function decorateHero() {
+  const existingEl = document.querySelector('.mobile-awareness-app-hero');
+  if (!existingEl) return;
+  let imageContainer = existingEl.querySelector('.responsive-image-swap');
+  if (!imageContainer) imageContainer = existingEl.querySelector('picture');
+  existingEl.removeChild(imageContainer);
+
+  const overlay = wrapContents(existingEl, { innerAttrs: { class: 'section-wrapper' } });
+  wrapContents(overlay, { innerAttrs: { class: 'app-hero-overlay' } });
+  overlay.parentElement.appendChild(imageContainer);
+}
+
+function decorateSignInReminder() {
+  const existingEl = document.querySelector('.mobile-awareness-app-sign-in-reminder');
+  if (!existingEl) return;
+  existingEl.classList.add('section-wrapper');
+
+  const contentBox = wrapContents(existingEl, { innerAttrs: { class: 'gray-zone' } });
+  hoist('picture', contentBox);
+
+  wrapContents(existingEl, { innerAttrs: { class: 'content' } });
+  const elements = contentBox.querySelectorAll('h2, p');
+  const beforeHeader = [];
+  for (const el of elements) {
+    if (['H2', 'H1', 'H3'].includes(el.nodeName)) break;
+    if (el.innerHTML) beforeHeader.push(el);
+  }
+  beforeHeader.forEach((el) => el.classList.add('image-caption'));
+  contentBox.querySelectorAll('p').forEach((p) => {
+    if (p.innerHTML.trim() === '') p.parentElement.removeChild(p);
+  });
+}
+
+function decorateTutorialPrompt() {
+  const existingEl = document.querySelector('.mobile-awareness-app-tutorial-prompt');
+  if (!existingEl) return;
+  existingEl.classList.add('section-wrapper');
+
+  const linkRow = existingEl.querySelector('.row');
+  const linkEls = linkRow.querySelectorAll(':scope > *');
+  const links = [];
+  for (const el of linkEls) {
+    links.push(el.innerHTML);
+  }
+  linkRow.parentElement.removeChild(linkRow);
+
+  const cardRow = existingEl.querySelector('.row');
+  const cardEls = hoist('.row > *', cardRow, false, existingEl.querySelector('.tutorial-prompts'));
+  cardRow.parentElement.removeChild(cardRow);
+
+  cardEls.forEach((cardEl, index) => {
+    cardEl.classList.add('tutorial-card');
+    hoist('picture', cardEl, true);
+    cardEl.querySelectorAll('p').forEach((p) => {
+      if (p.innerHTML.trim() === '') p.parentElement.removeChild(p);
+    });
+
+    const url = links[index];
+    if (url) {
+      const newEl = replaceTag(cardEl, 'a');
+      newEl.setAttribute('href', url);
+    }
+  });
+
+  const link = createTag('span', { class: 'fake-link no-mobile no-tablet' });
+  link.innerHTML = 'Text me a download link >';
+  openModalLink(link, /.*\/([^/]*)\/?$/gi.exec(window.location.href.split('?')[0])?.[1]);
```

Comment:
@davidnuescheler suggested we avoid optional chaining for better compatibility with older browsers (only ~92% available) - I think there are a few cases in this PR